.goreleaser.yml

```diff
@@ -10,11 +10,11 @@ env:
 before:
   hooks:
     - go mod tidy
-    - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
-# if running a release we will generate the images in this step
-# if running in the CI the CI env va is set by github action runner and we dont run the ko steps
-# this is needed because we are generating files that goreleaser was not aware to push to GH project release
-    - /bin/bash -c 'if [ -n "$CI" ]; then make sign-container-release; fi'
+    # - /bin/bash -c 'if [ -n "$(git --no-pager diff --exit-code go.mod go.sum)" ]; then exit 1; fi'
+    # if running a release we will generate the images in this step
+    # if running in the CI the CI env va is set by github action runner and we dont run the ko steps
+    # this is needed because we are generating files that goreleaser was not aware to push to GH project release
+    # - /bin/bash -c 'if [ -n "$CI" ]; then make sign-container-release; fi'
```

Comment:
Just to double check, this is not needed?

Reply:
it is, my bad

Reply:
did a small refactor